MvvmCross-Forms/MvvmCross.Forms/Views/IMvxFormsViewPresenter.cs

```diff
@@ -1,11 +1,15 @@
 using System;
+using MvvmCross.Core.Views;
 using MvvmCross.Forms.Platform;
 
 namespace MvvmCross.Forms.Views
 {
-    public interface IMvxFormsViewPresenter
+    public interface IMvxFormsViewPresenter : IMvxAttributeViewPresenter
```

Comment:
Is the name "IMvxFormsViewPresenter" too close to "IMvxFormsViewPresenter"? Perhaps something like "IMvxFormsNativeViewPresenter" or "IMvxFormsPlatformViewPresenter" or inline with Forms naming "IMvxFormsOnPlatformViewPresenter"

Reply:
We already have this interface and it is implement on all platforms. So not to break the naming i kept it the same.